src/controllers/dateOfBirthController.ts

```diff
@@ -0,0 +1,71 @@
+import { NextFunction, Request, Response } from "express";
+import * as config from "../config";
+import { validationResult } from "express-validator";
+import { formatValidationError, getPageProperties } from "../validations/validation";
+import { BASE_URL, DATE_OF_BIRTH, EMAIL_ADDRESS, WHEN_IDENTITY_CHECKS_COMPLETED } from "../types/pageURL";
+import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../utils/localise";
+import { Session } from "@companieshouse/node-session-handler";
+import { USER_DATA } from "../utils/constants";
+import { ClientData } from "model/ClientData";
+import { saveDataInSession } from "../utils/sessionHelper";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const previousPage: string = addLangToUrl(BASE_URL + EMAIL_ADDRESS, lang);
+    const currentUrl: string = BASE_URL + DATE_OF_BIRTH;
+
+    const clientData: ClientData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
+    saveDataInSession(req, USER_DATA, clientData);
+    let payload;
+    if (clientData.dateOfBirth) {
+        const dateOfBirth = new Date(clientData.dateOfBirth);
```

Comment:
When the first time page will load will this dateOfBirth in clientData be not null?